cloud/google/instancestatus.go

```diff
@@ -35,8 +38,11 @@ func (gce *GCEClient) instanceStatus(machine *clusterv1.Machine) (instanceStatus
 
 // Sets the status of the instance identified by the given machine to the given machine
 func (gce *GCEClient) updateInstanceStatus(machine *clusterv1.Machine) error {
+	if gce.v1Alpha1Client == nil {
```

Comment:
Rather than fail silently It seems like it would be better to return an error if you don't have a client available to you here